src/chain/types.hpp

```diff
@@ -47,16 +47,20 @@ enum class SignerStatus { UNKNOWN, ERROR, GOOD };
 
 using sha256 = std::vector<uint8_t>;
 using bytes = std::vector<uint8_t>;
-using timestamp = std::string;
+using timestamp_type = uint64_t;
 using block_height_type = std::string;
 
 using transaction_id_type = bytes;
 using requestor_id_type = sha256;
 using merger_id_type = uint64_t;
 using signer_id_type = uint64_t;
 using transaction_root_type = sha256;
+using block_header_hash_type = sha256;
+using block_id_type = sha256;
 using chain_id_type = sha256;
```

Comment:
```suggestion
using chain_id_type = uint64_t;
```
chain_id는 GENTEST1 (8 bytes) 입니다

Reply:
@dos-gruut 
수정했습니다. 말씀하신 'GENTEST1'는 나중에 config 파일 만들때 한꺼번에 수정하겠습니다.
임시로 1로 할당했습니다.